content/docs/command-reference/add.md

```diff
@@ -248,36 +248,42 @@ For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
 
 ## Example: Dvcignore
```

Comment:
Hey @sarthakforwet I see you use the same commit message over and over in all your commits. This isn't a good practice, please check out our contribution guide again, there are guidelines about this. Thanks

Reply:
Hmm I guess we don't have it in the docs contrib guide but it's in the core contrib guide here: https://dvc.org/doc/user-guide/contributing/core#commit-message-format-guidelines

You can also see the messages in any of my commits, they all have some structure and context info. Sometimes the commit message is longer than the change itself.

Reply:
Okay, will remember it from next time. 😄

Reply:
Thanks!